src/core_plugins/elasticsearch/lib/__tests__/routes.js

```diff
@@ -1,25 +1,31 @@
 import { format } from 'util';
 
 import * as kbnTestServer from '../../../../test_utils/kbn_server';
+import { esTestCluster } from '../../../../test_utils/es';
 
 describe('plugins/elasticsearch', function () {
   describe('routes', function () {
     let kbnServer;
+    const es = esTestCluster.use({
+      name: 'core_plugins/es/routes',
+    });
 
     before(async function () {
-      // Sometimes waiting for server takes longer than 10s.
+      // Sometimes waiting for kbnServer/es takes longer than 10s.
       // NOTE: This can't be a fat-arrow function because `this` needs to refer to the execution
       // context, not to the parent context.
-      this.timeout(60000);
+      this.timeout(3 * 60 * 1000);
```

Comment:
It might have to download es on a slow network connection (like mine, at the hotel I'm staying at)